cli/tests/init.bats

```diff
@@ -216,10 +216,10 @@ EOF
   echo "requests" > requirements.txt
   [ ! -e .flox ] || "$FLOX_BIN" delete -f
   "$FLOX_BIN" init --auto-setup --name "$NAME"
-  FLOX_SHELL="bash" USER="$REAL_USER" NO_COLOR=1 run -0 expect "$TESTS_DIR/init/python-env.exp" "$PROJECT_DIR"
-  assert_line --partial "Activating python virtual environment"
-  assert_line --partial "deactivate is a function"
-  assert_line --partial "deactivate ()"
+  "$FLOX_BIN" activate -- true # populate the virtual environment
+  run bash -c 'VIRTUAL_ENV="$PWD/.flox/cache/python" PATH="$VIRTUAL_ENV/bin:$PATH" python3 -m pip show requests'
```

Comment:
This no longer tests that `activate` is being sourced by the `profile` scripts, in addition to `on-activate`, which I think was the original intention from #1299. Is there a benefit to setting up the venv manually like this rather than calling `flox activate -- python3 -m pip show requests`?

Reply:
I totally misinterpreted what this test was doing. You can replace `expect` and still test that the scripts are working correctly via `flox activate -- type deactivate` which is essentially what the original `expect` test was doing.